packages/cursorless-vscode-e2e/src/suite/scopeVisualizer/runBasicMultilineContentTest.ts

```diff
@@ -0,0 +1,84 @@
+import { openNewEditor } from "@cursorless/vscode-common";
+import * as vscode from "vscode";
+import { checkAndResetFakes } from "./checkAndResetFakes";
+import { injectFakes } from "./injectFakes";
+import { ExpectedArgs } from "./scopeVisualizerTest.types";
+
+/**
+ * Tests that the scope visualizer works with multiline content, by
+ * ensuring that the correct decorations are applied so that it looks
+ * like `./runBasicMultilineContentTest.png`.
```

Comment:
Annoyingly, we can't embed this image or link to it because then our docs build breaks 🙄.  See https://github.com/cursorless-dev/cursorless/issues/1254

Reply:
Yeah really annoying